hydroengine_service/main.py

```diff
@@ -1470,6 +1474,75 @@ def get_metocean_data():
         mimetype='application/json'
     )
 
+@app.route('/get_windfarm_data', methods=['POST'])
+@flask_cors.cross_origin()
+def get_windfarm_data():
+    r = request.get_json()
+
+    features = r['features']
+    collection = ee.FeatureCollection(features)
+
+
+    ports = ee.FeatureCollection("users/fbaart/worldlogistic/port")
+    gebco = ee.Image("projects/dgds-gee/gebco/2019")
+    coast = ee.Image("users/gena/land_polygons_image")
+    wind = ee.Image("projects/dgds-gee/gwa/gwa3/10m").rename('wind_magnitude_mean')
+
+    scale = 1000
```

Comment:
fixing scale might be an issue, might be better to pass as request parameters

Reply:
I want this specific at this scale.